jfr-daemon/build.gradle.kts

```diff
@@ -41,7 +41,7 @@ tasks.shadowJar {
 
     manifest {
         attributes(
-                // Agent-Class ?
+                "Agent-Class" to "com.newrelic.jfr.daemon.agent.AgentMain",
```

Comment:
I don't think this should cause any issues when the jdr-daemon is embedded into the Java agent but we'll want to test and verify that assumption.